release.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Can you change it to `#!/bin/sh`

Some systems do not have `bash` installed, and the only bash-specific thing is `-n1 key` parameter to `read`